horovod/spark/common/store.py

```diff
@@ -164,6 +165,15 @@ def read(self, path):
         with self.get_filesystem().open(self.get_localized_path(path), 'rb') as f:
             return f.read()
 
+    def read_serialized_keras_model(self, ckpt_path, model=None):
+        """
+        This function will be overridden by DBFSLocalStore.
```

Comment:
* First sentence should describe what this method does.
* Describe the params and the return value.